### PR TITLE
Members: fix mobile UI issues in members page

### PR DESCRIPTION
**Pull request by Pineirin**
* closes: #646

Screenshot from 2022-05-16 09-56-03
Screenshot from 2022-05-16 09-56-10

### CODE REVIEW

File: invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMembersResultItem.js [hunk 1]
```diff
@@ -139,7 +139,6 @@ export class ManagerMembersResultItem extends Component {
               )}
               {result.permissions.can_delete && (
                 <Button
-                  fluid
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMembersResultItem.js:142]: this is breaking the display of the button in the desktop view

File: invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMembersResultItem.js [hunk 2]
```diff
@@ -68,13 +68,13 @@ export class ManagerMembersResultItem extends Component {
...
                   >
-                    <b>{result.member.name}</b>
+                    <b className="mr-10">{result.member.name}</b>
 
                     {result.member.is_group && (
-                      <Label className="ml-10">{i18next.t("Group")}</Label>
+                      <Label className="mr-10">{i18next.t("Group")}</Label>
                     )}
                     {result.is_current_user && (
-                      <Label color="blue" className="ml-10">
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMembersResultItem.js:77]: this is breaking the display of the button in the desktop view

File: invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMembersResultItem.js [hunk 3]
```diff
@@ -155,6 +160,7 @@ export class ManagerMembersResultItem extends Component {
               {result.permissions.can_delete && (
                 <Button
                   fluid
+                  className={"non-fluid-mobile"}
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMembersResultItem.js:163]:
```suggestion
                  className="non-fluid-mobile"
```

**Comment by Pineirin** [comment]
Extended in: https://github.com/inveniosoftware/invenio-communities/pull/671

**Pull request closed without merging**